APIManager.js

```diff
@@ -0,0 +1,91 @@
+//This is the class that will manage all your APIs
+
+class APIManager {
+    constructor() {
+        this.data = {
+            user: {},
+            friends: [],
+            quote: '',
+            pokemon: {},
+            about: ''
+        }
+    }
+
+    getUser() {
+        $.ajax({
+            method: 'GET',
+            url: 'https://randomuser.me/api/',
+            success:  (users) => {
+                this.data.user.fName = users.results[0].name.first
+                this.data.user.lName = users.results[0].name.last
+                this.data.user.city = users.results[0].location.city
+                this.data.user.country = users.results[0].location.country
+                this.data.user.imagine = users.results[0].picture.thumbnail
+            }
+        })
+    }
+
+    getFriends() {
+            $.ajax({
+                method: 'get',
+                url: 'https://randomuser.me/api/?results=6',
+                success: (friend) => {
+                    this.data.friends=[]
+                    let newArr = friend.results.map( f => f.name.first + " " + f.name.last)
+                    this.data.friends.push(newArr)
```

Comment:
I wouldn't push an array into another array. This can create problems down the road. This whole method can be one line:
`this.friends = friend.results.map( f => f.name.first + " " + f.name.last)`